<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageEditorCodec</class>
 <widget class="QWidget" name="PageEditorCodec">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="listCodecs"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlayButtons">
     <property name="spacing">
      <number>12</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnCheckAll">
       <property name="text">
        <string>Check all</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnUcheckAll">
       <property name="text">
        <string>Uncheck all</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hspacerButtons">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
